src/site/_includes/spaces.njk

```diff
@@ -6,7 +6,16 @@ layout: default-next
   <div class="card__header">
     <div class="avatars">
     {% for host in space.data.hosts %}
-      <img class="avatar" src="{{ host.image }}" alt="alt" decoding="async" height="40" loading="lazy" width="40">
+      {% set alt = "Profile image for " + collections.authors[host].title | i18n(locale) %}
+      {% Img
+        src=collections.authors[host].image,
+        class="avatar",
+        width="96",
```

Comment:
I don't know why the original source had 40px hardcoded here, but the CSS assigns a height/width of 96px, so I'm using that here.